Jenkinsfile

```diff
@@ -1,10 +1,5 @@
 #!groovy
-@Library('waluigi@v3.1.1') _
+@Library('waluigi@v6.0.0') _
```

Comment:
I guess this should be bumped to 6.0.1 now?

Reply:
Yeah I still need to go through and update all existing PRs to use 6.0.1. As I've been in meetings today I haven't had a chance yet sorry.

Reply:
```suggestion
@Library('waluigi@v6.0.1') _
```